#! /bin/sh /usr/share/dpatch/dpatch-run
## 20_man.files.typo.dpatch by Stephen Gran <sgran@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Better explain get/set options

@DPATCH@
--- a/hdparm.8	(revision 314)
+++ b/hdparm.8	(working copy)
@@ -15,6 +15,8 @@
 When no flags are given,
 .I -acdgkmnru
 is assumed.
+For Get/Set options, a query without an optional parameter (e.g., \-d) will query (get)
+the drive state, and with a parameter (e.g., \-d0) will set the drive state.
 .TP
 .I -a 
 Get/set sector count for filesystem (software) read-ahead.
